CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - [#90] Capture HTTP response headers in django and flask.
 
 ### Changed
+- A recording is now created atomically. It is first written to a temp file, which is then
+  renamed to the final file.
```

Comment:
This should go under *unreleased* I think. 
(Git doesn't do a great job merging changelogs, it's good to keep an eye on these.)

Reply:
Oops, yeah. Fixed.